examples/top_pods.js

```diff
@@ -1,39 +1,35 @@
-const k8s = require('../dist/index');
+const k8s = require('@kubernetes/client-node');
```

Comment:
this needs to be removed b/c otherwise we are testing against the published package vs `HEAD`

Reply:
Most of the examples use this. I will updated the others to use `HEAD` to match your comment.

Reply:
Now I'm actually conflicted :) Perhaps it is better to have a working example? The trouble with not using `HEAD` is that if we change the API then we don't have continuous validation that the examples are working...

Maybe we can add a comment to each of these that says:
`// in a real program use require('@kubernetes/client-node')`

or something like that.

Reply:
Ok, the rest of these seem to use `require('@kubernetes/client-node');`. I will make them use `require('../dist/index');` but maybe it would be helpful to have a comment on how to import the package? Or maybe the readme does that already. Thinking out loud here. For now I will just import them.